data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -45,6 +45,7 @@ def __init__(self):
         self.persistent_poverty_df: pd.DataFrame
         self.census_decennial_df: pd.DataFrame
         self.census_2010_df: pd.DataFrame
+        self.national_tract_df: pd.DataFrame
```

Comment:
We might want to drop the fields that aren't used in the score from this, just so we don't have to continue the support. persistent poverty, child opportunity index i think can be removed. wdyt?

Reply:
Want me to do that here, or in the cleanup ticket? I kinda just want to keep focusing on correctness tests and then move to cleanup when I have a nice test suite under me, but I don't feel strongly.

Reply:
I put this into the cleaning ticket. I definitely think "get it right, get it tight" is the right approach here!

Reply:
I removed it in my tribal PR, so that should be taken care once we merge to Narwahl Release